client/src/components/settings/Certification.js

```diff
@@ -415,6 +415,11 @@ export class CertificationSettings extends Component {
       { types: {} }
     );
 
+    const formFields = challengeTitles.reduce((formFields, current) => {
+      formFields.push({ field: current, fieldLabel: current });
```

Comment:
Are you sure you want the `name` attribute to be the same as the label text?  You assigned `current` to `field` and `fieldLabel`.

Reply:
To be honest, no, I am not sure. I thought, seeing as this has to do with the challenge titles, it does not matter which is under which, except for whatever is under `fieldLabel` will be displayed.

I have claimed certificates, and been on all of the certificate-related pages, and have yet to see any breaks due to this.

Reply:
I can see in production that the name and label are the same.